source/numerics.tex

```diff
@@ -8023,10 +8023,10 @@
 the
 \tcode{valarray}
 template, which is one-dimensional.
-The set of one-dimensional index values specified by a
-\tcode{gslice}
-are $$k = s + \sum_ji_jd_j$$
-% \$k = s + sum from j \{ i sub j d sub j \}\$
+The set of one-dimensional index values specified by a \tcode{gslice} are
+
```

Comment:
Wait, we didn't use to have paragraph breaks here originally, why did you introduce them?